Decred Wallet/Features/History/TransactionHistoryViewController.swift

```diff
@@ -231,32 +231,23 @@ extension TransactionHistoryViewController: UITableViewDataSource, UITableViewDe
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         let cell = self.tableView.dequeueReusableCell(withIdentifier: TransactionTableViewCell.identifier) as! TransactionTableViewCell
-        if self.Filtercontent.count != 0 {
-            let transaction = Filtercontent[indexPath.row]
-            cell.setData(transaction)
+        guard filteredTransactions.count != 0 else {
+            cell.setData(allTransactions[indexPath.row])
             return cell
         }
-        
+        cell.setData(filteredTransactions[indexPath.row])
         return cell
     }
     
-    func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
-        if self.Filtercontent.count > indexPath.row {
-            if self.Filtercontent[indexPath.row].Animate {
-                cell.blink()
-            }
-            self.Filtercontent[indexPath.row].Animate = false
-        }
-    }
-    
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         let storyboard = UIStoryboard(name: "TransactionFullDetailsViewController", bundle: nil)
         let subContentsVC = storyboard.instantiateViewController(withIdentifier: "TransactionFullDetailsViewController") as! TransactionFullDetailsViewController
-        if self.Filtercontent.count == 0{
-            return
+        
+        if filterActive {
```

Comment:
We can do away with this `filterActive` variable and instead use `self.filteredTransactions.count > 0`.

Reply:
also removed ` self.filterActive = currentFilter != DcrlibwalletTxFilterAll` and the `self.filterActive` variable as it is no longer needed.

Reply:
fixed in e594317